Update mail.md (#10151)
laravel/docs: mail.md

composer require symfony/mailgun-mailer symfony/http-client
```

> Make two changes in your application's `config/mail.php` configuration file.  First, change default MAIL_MAILER default value to `mailgun` 

Reviewer: You might want to format this as code:
```suggestion
Make two changes in your application's `config/mail.php` configuration file.  First, change default `MAIL_MAILER` default value to `mailgun` 
```

Author: That sounds good.  I am a dev in Detroit - not a Github super expert.

> Make two changes in your application's `config/mail.php` configuration file.  First, change default MAIL_MAILER default value to `mailgun` 
> 
>     'default' => (env('MAIL_MAILER', 'mailgun'),

Reviewer: You could use fenced code blocks here and display the filename directly on the code block itself:
````suggestion
Make two changes in your application's mail configuration file.  First, change default MAIL_MAILER default value to `mailgun` 

```php filename=config/mail.php
    'default' => (env('MAIL_MAILER', 'mailgun'),
```
````

Author: lgtm.  I didn't know how to do that - I don't specialize in Github and MD

Reviewer: Not a problem, we all never stop learning :)

> Make two changes in your application's `config/mail.php` configuration file.  First, change default MAIL_MAILER default value to `mailgun` 
> 
>     'default' => (env('MAIL_MAILER', 'mailgun'),

Reviewer: This is probably a typo:
```suggestion
Make two changes in your application's `config/mail.php` configuration file.  First, change default MAIL_MAILER default value to `mailgun` 

    'default' => env('MAIL_MAILER', 'mailgun'),
```

Author: yes

> Make two changes in your application's `config/mail.php` configuration file.  First, change default MAIL_MAILER default value to `mailgun` 
> 
>     'default' => (env('MAIL_MAILER', 'mailgun'),
> 
> Second, add the following configuration array to your array of `mailers`:

Reviewer: Why not make it a list while we are at it?
```suggestion
Make two changes in your application's `config/mail.php` configuration file:

1. Change default MAIL_MAILER default value to `mailgun` 

    'default' => (env('MAIL_MAILER', 'mailgun'),

2. Add the following configuration array to your array of `mailers`:
```

Author: yes
